core/src/mindustry/net/Administration.java

```diff
@@ -478,6 +478,7 @@ private void load(){
         autosave("Whether the periodically save the map when playing.", false),
         autosaveAmount("The maximum amount of autosaves. Older ones get replaced.", 10),
         autosaveSpacing("Spacing between autosaves in seconds.", 60 * 5),
+        locale("Default locale in server", "default"),
```

Comment:
This doesn't seem to do anything.

Reply:
Use for server(headless). can't set locale in ui

Reply:
But it doesn't actually do anything, since the locale is never used anywhere in the vanilla server. `Config.locale` is *never read*. If you want a system for defining custom configs for plugins on the server, I can add that, but there's no sense adding a configuration option that does nothing.

Reply:
![Snipaste_2022-04-17_22-08-02](https://user-images.githubusercontent.com/15688938/163718099-93ff0c28-dbb6-4e84-a7c2-a8d49c41bb63.png)
`Vars.loadSettings` read this config

Reply:
That's... really indirect and hacky, you're relying on implementation details to get that, instead of the Config API. There's no validation or error handling on the locale string, either.